gst/nnstreamer/tensor_filter/tensor_filter.c

```diff
@@ -1112,10 +1112,11 @@ nnstreamer_filter_async_output_callback (void *async_handle,
   GstTensorFilterProperties *prop = NULL;
   GstBuffer *outbuf = NULL;
 
-  g_return_if_fail (output != NULL);
-  g_return_if_fail (async_handle != NULL);
+  g_return_val_if_fail (data != NULL, -EINVAL);
+  g_return_val_if_fail (info != NULL, -EINVAL);
```

Comment:
```info``` is not used. necessary?

Reply:
unnecessary for tensor-filter, but should be passed in the callback :)